### PR TITLE
Arc extension: update readme and icon

### PR DESCRIPTION
**Pull request by rgrunberg**
## Description
* Update the readme to include a link to download Arc
* Update the extension icon with an Arc brand asset

## Screencast
CleanShot 2023-09-12 at 13 03 10@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Visual-Studio-Coder** [comment]
I'm okay with everything but the icon change. I think they worked with arc browser to make it.

File: extensions/arc/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Arc
 
-Search and quickly navigate Arc's history and open tabs. You can open recent tabs in Arc or Little Arc as well as copy the websites to quickly share them.
+Search and quickly navigate Arc's history and open tabs. You can open recent tabs in Arc or Little Arc as well as copy websites to quickly share them.
+
+Experience a calmer, more personal internet in this browser designed for you. Let go of the clicks, the clutter, the distractions with the Arc browser. If you don't have Arc yet, download it at [arc.net](https://arc.net).
```
Comment (danpalmer) [inline extensions/arc/README.md:5]: Not sure how Raycast folks think about the extensions marketplace, but personally I don't feel this is the right place to market the external piece of software. I suspect that discovering new software to use via extensions is not a common use case. I also just feel that the strong marketing message feels out of place in this context.

**Review by danpalmer — COMMENTED** [review]
I'm not sure this is right to merge. The grammar fix in the first paragraph of copy seems good though!

**Comment by rgrunberg** [comment]
> I'm okay with everything but the icon change. I think they worked with arc browser to make it.

I work at The Browser Company and this icon is from our brand designer :)

File: extensions/arc/README.md (hunk 1 above)
Comment (rgrunberg) [reply extensions/arc/README.md]: Changed this to just link out from the existing first paragraph

**Comment by Visual-Studio-Coder** [comment]
@rgrunberg just keep the icon the same.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me @rgrunberg 👏

**Comment by danulqua** [comment]
Not a fan of a new icon...

**Pull request merged**